packs/mlops/dagshub/dashboards/dagshub.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "DagsHub model training example",
+  "description": "Dashboard example of the Loss metric monitored to DagsHub",
+  "pages": [
+    {
+      "name": "DagsHub model training example",
+      "description": "",
+      "widgets": [
+        {
+          "visualization": {
+            "id": "viz.line"
+          },
+          "layout": {
+            "column": 1,
+            "row": 1,
+            "height": 3,
+            "width": 12
+          },
+          "title": "",
+          "rawConfiguration": {
+            "legend": {
+              "enabled": true
+            },
+            "nrqlQueries": [
+              {
+                "accountId": 3240610,
+                "query": "SELECT latest(loss) FROM Metric TIMESERIES SINCE 1800 seconds ago"
```

Comment:
(disclaimer: I don't know anything about DagsHub)
Will this `loss` event be available for every user that sets up DagsHub?
The goal of the dashboard is that it should work for every user who tries this out.

Are there other charts that could be added to make this a compelling dashboard?
Or does this really depend on the customer's use case? Maybe in that case, would it make sense to use a markdown widget on the dashboard and provide some more info?